flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/FineGrainedSlotManager.java

```diff
@@ -629,6 +629,24 @@ private void checkResourceRequirements() {
         }
 
         LOG.info("Matching resource requirements against available resources.");
+        LOG.info("Missing resources:");
```

Comment:
Don't you want to use StringBuilder and log it as one record? 
These records from the current solution could be shuffled with other logs and as a result, it will be many lines on different orders and it won't be clear what line related to what.

Reply:
sounds good